### ISSUE TITLE
NUM 13:32 "eats up inhabitants" = "is very large"?

### ISSUE THREAD

**Issue by SusanQuigley** [Door43/udb-en#132]
ULB The land that we looked at is a land that **eats up its inhabitants**. 
UDB The land that we explored is **very large**. We cannot conquer it.

The UDB translation comes from the TFT. I see various interpretations of "that eats up its inhabitants" but other than the TFT, I don't see any that say that it means that the land is vey large.

Could the UDB fit more with the commentaries?
"The land that we explored is a dangerous land"

Various interpretations: it was infertile so people would die of hunger,  there were plagues, there was civil war, there were threats from neighboring enemies,

**Comment by TomWarren** [comment]
The UDB NUM 14:32-33 now reads: 
\v 32 So those men gave to the Israelite people a bad report about the land that they had explored. They said, "The land that we explored destroys the life those who try to attack it. And all the people living there are very tall. 
\v 33 We also saw giants there. They are the descendants of Anak (who come from the Nephilim, those giant people). When we saw these giants we felt as small as grasshoppers and they thought that we looked like grasshoppers too!"

Hope that helps.
